_build/bindep.txt

```diff
@@ -8,3 +8,5 @@ git-lfs [platform:rpm]
 sshpass [platform:rpm]
 rsync [platform:rpm]
 epel-release [platform:rpm]
+unzip [platform:dpkg]
```

Comment:
```suggestion

```
This line is unnecessary, AWX-EE isn't a debian-based platform

AWX-EE will only match with these bindep platform profiles:
```shell
docker run --user="root" -it quay.io/ansible/awx-ee:latest bash
pip install bindep
bindep --profiles
```
```text
Platform profiles:
platform:base-py3
platform:centos
platform:centos-9
platform:redhat
platform:rpm
```